bom/runtime/pom.xml

```diff
@@ -176,6 +176,7 @@
         <logstash-gelf.version>1.13.0</logstash-gelf.version>
         <jsch.version>0.1.55</jsch.version>
         <jzlib.version>1.1.1</jzlib.version>
+        <unboundid-ldap.version>4.0.13</unboundid-ldap.version>
```

Comment:
It looks like this is only used for testing? If so it should be in the build parent rather than the bom.

Reply:
Makes sense. Fixed.